src/main/java/nextstep/ladder/model/Player.java

```diff
@@ -0,0 +1,58 @@
+package nextstep.ladder.model;
+
+import java.util.Objects;
+
+public class Player {
+    private final Name name;
+    private final Position position;
```

Comment:
Player 클래스도 불변으로 작성해주셨네요 👍 
비즈니스를 구현할 때, 크게 2가지의 class 가 존재한다고 생각하는데요.

1. 상태가 변경되는 객체
2. 수학적인 값을 나타내는 객체

지금 구현하신것을 보면, Player 는 수학적인 값을 나타내는 값객체로 만드신것으로 보여요! 
어떤 생각을 가지고 이렇게 구현하셨는지 궁금합니다!

개인적인 생각으론 Player 는 상태가 변경되는 객체라고 생각해요.
만약 pobi 플레이어의 위치가 변한다면, 그 플레이어는 pobi 가 아닐까요? 🤔

Reply:
“수학적인 값을 나타내는 객체”라는게 “불변객체”로 이해가 됩니다만,
오버엔지니어링으로 멀티 스레드 생각하고 "직접 상태 값을 변경하는 건 위험하지 않나?"라는 생각으로 코드 작성했던 거 같습니다

다시 생각해본다면 과제와 같은 단일 스레드에서 
말씀해주신거 같이 상태를 변경하더라도 문제가 일어날거 같지 않고
오히려 불변객체를 만듦으로써 리소스 낭비가 발생했다는 생각도 듭니다

Reply:
```suggestion
public class Player {
    private final Name name;
    private Position position;
```

이렇게 변경이 가능한 Position 을 갖고 있는 Player로 구현해야한다는 피드백이었습니다 😅

Pobi 가 1을 움직였다.
Pobi 가 다시 2를 움직였다.

여기서 플레이어 Pobi는 동일하고, Position 만 이동하였으며 Position은 위치가 1에서 3으로 변경되었으니 값객체로써 새롭게 할당하는건 동의하는바입니다 😄